src/pages/index.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import { PageProps } from 'gatsby';
+import { MainPage } from '~/components/main-page/main-page';
+import { SEO } from '~/partials';
+
+const Homepage: React.FC<PageProps> = () => <MainPage />;
```

Comment:
```suggestion
const MainPage: React.FC<PageProps> = () => <MainPage />;
```

Reply:
Const main page which exports MainPage?

Reply:
then name it as just Page, we don't want to have two names for same page